traffic-policy/variables/conn.ip-intel.categories.mdx

```diff
@@ -23,647 +23,181 @@ Or you can block all anonymous proxies which would include Tor as well as any ot
+| `blocklist.net.emergingthreats.compromised_ips` | From [emergingthreats.net](https://rules.emergingthreats.net/blockrules/): Emerging Threats Intelligence from Proofpoint provides information on IP addresses and domains that may be compromised.                                                                                                                                                                                                    | [Source](https://rules.emergingthreats.net/blockrules/compromised-ips.txt) |
+| `blocklist.org.firehol.abusers_1d`              | From [firehol.org](https://iplists.firehol.org/?ipset=firehol_abusers_1d): An ipset made from blocklists that track abusers in the last 24 hours.                                                                                                                                                                                                                                                     | [Source](https://iplists.firehol.org/files/firehol_abusers_1d.netset)      |
+| `blocklist.org.firehol.abusers_30d`             | From [firehol.org](https://iplists.firehol.org/?ipset=firehol_abusers_30d): An ipset made from blocklists that track abusers in the last 30 days.                                                                                                                                                                                                                                                     | [Source](https://iplists.firehol.org/files/firehol_abusers_30d.netset)     |
+| `blocklist.org.firehol.level_1`                 | From [firehol.org](https://iplists.firehol.org/?ipset=firehol_level1): The objective is to create a blacklist that can be safe enough to be used on all systems, with a firewall, to block access entirely, from and to its listed IPs. The key prerequisite for this cause, is to have no false positives. All IPs listed should be bad and should be blocked, without exceptions.                   | [Source](https://iplists.firehol.org/files/firehol_level1.netset)          |
+| `blocklist.org.firehol.level_2`                 | From [firehol.org](https://iplists.firehol.org/?ipset=firehol_level2): An ipset made from blocklists that track attacks, during the last 48 hours.                                                                                                                                                                                                                                                    | [Source](https://iplists.firehol.org/files/firehol_level2.netset)          |
```

Comment:
I'd like to see these descriptions removed from this table, but I've created a separate linear issue for that instead of slowing this PR down:
https://linear.app/ngrok/issue/DOC-12/remove-unnecessary-descriptions-from-the-ip-intel-var-docs